test/fixtures/misc-templates/deployment-w-progress.yml

```diff
@@ -0,0 +1,29 @@
+apiVersion: extensions/v1beta1
```

Comment:
Oh true, I probably should have suggested `test/fixtures/invalid/bad_probe.yml` instead.